Game.java

```diff
@@ -206,32 +243,37 @@ protected static boolean isFull(int[][] table){
 		}
 		return true;
 	}
-	protected boolean isFull() {
-		// Check wether there are no moves yet
-		return isFull(table);
-	}
 	public int step(String move) {
```

Comment:
I would change this method name to be `humanStep`

Reply:
But it also calls computer step too.